examples/index.html

```diff
@@ -157,7 +159,7 @@ <h3>Translating texts</h3>
         tStatusNoResults: () => '',
         tStatusSelectedOption: (selectedOption, length) => '',
         tStatusResults: () => '' }</code> options.</p>
-      <label for="autocomplete-i18n">Select your country</label>
+      <label for="autocomplete-i18n">Country</label>
       <div id="tt-i18n" class="autocomplete-wrapper"></div>
 
       <h3>Multiselect</h3>
```

Comment:
Consider updating the labels in this section to "Country" from "Select your country" to match the updates on the base component.